### ISSUE TITLE
Add to .env file "max_detect_size" option

### ISSUE THREAD

**Issue by pospielov** [exadel-inc/CompreFace#609]
To improve performance on huge images, we resize it before face detection. The downside of this is that small faces could be not recognized. 

Some users complain that we do not catch too small faces in their 1080p camera.

We need to make resizing a configurable variable, so users choose if we need to resize at all, or what resize to apply

Here is where we init the variable in Python:
https://github.com/exadel-inc/CompreFace/blob/c44c92eaea31798f8e68c91acc36144d3b8ab411/embedding-calculator/src/constants.py#L24

So basically all we need is to add the "IMG_LENGTH_LIMIT" environment variable to `docker-compose` and then add it to `.env` file

**Comment by deepto98** [comment]
@pospielov Hi, I'd like to work on this issue, can you assign this to me?

**Comment by pospielov** [comment]
Hi, thanks for the interest in our repository, I assigned the issue to you

**Comment by sheetal2001p** [comment]
Hi ,I just want to start my open source contribution through hactoberfest but I am a beginner and don't know much about how can I contribute!! so it is the issue for beginners right? So please can anyone help me how can I able to solve this issue ? Is it necessary to be assigned to me?

**Comment by pospielov** [comment]
Hi, thanks for the interest in our repository.
I assign tasks to people because there is no reason two people work on one issue in parallel.
Please, check the list of issues that are not assigned to anybody.
Also, I'll try to add more issues for beginners during a couple of days.

**Comment by Onyinye91-ctrl** [comment]
@pospielov is there any first time issues you can assign to me?

**Comment by Onyinye91-ctrl** [comment]
Thank You @sheetal2001p  @pospielov
